library/tasks/SequenceDataWithMoleculeTagMetrics.wdl

```diff
@@ -47,9 +47,10 @@ task CalculateGeneMetrics {
 task CalculateCellMetrics {
   input {
     File bam_input
+    File original_gtf
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.7"
+    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:kmk-mt-counts"
```

Comment:
can you rename the docker with a version number?

Reply:
Yes, the version number is changed.